packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -137,6 +162,9 @@ export class PHPRequestHandler {
 	#cookieStore: HttpCookieStore;
 	rewriteRules: RewriteRule[];
 	processManager: PHPProcessManager;
+	getFileNotFoundAction: FileNotFoundGetActionCallback = () => ({
+		type: '404',
```

Comment:
The PHPRequestHandler default file-not-found action is a simple 404 response because we can default to WordPress-specific handling in `bootWordPress()`.